### DISCUSSION TITLE
storage call error 0: Rust panic

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9209] [Storage Provider] [answered]
Crash, how can the following errors suddenly appear:
   computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: xxxx-System-Product-Name]: Rust panic: no unwind information

**Comment by TippyFlitsUK** [comment, marked as answer]
Hi @ChaiWenyu1983 :wave:

This error can sometimes occur when a sector's sealing process has been interrupted for whatever reason. Without additional logs from you, it is very difficult to work out why this might have happened.

If this persists, please [create a new issue using the provided template](https://github.com/filecoin-project/lotus/issues/new/choose) and provide all the requested information.

Many thanks
